src/rp2_common/pico_flash/flash.c

```diff
@@ -145,7 +145,18 @@ static int default_enter_safe_zone_timeout_ms(__unused uint32_t timeout_ms) {
         uint core_num = get_core_num();
         // create at low priority on other core
         TaskHandle_t task_handle;
+
+        // when FreeRTOS dynamic allocation is disabled (configSUPPORT_DYNAMIC_ALLOCATION == 0), the following instruction fails
+#ifndef configSUPPORT_DYNAMIC_ALLOCATION
```

Comment:
should be `if !configSUPPORT_DYNAMIC_ALLOCATION`

Reply:
it was also backwards

Reply:
How did I miss that? good catch!